pkg/testrunner/testrunner_test.go

```diff
@@ -10,10 +10,6 @@ func Test_Mutate_Validate_qos(t *testing.T) {
 	testScenario(t, "/test/scenarios/other/scenario_mutate_validate_qos.yaml")
 }
 
-func Test_disallow_root_user(t *testing.T) {
-	testScenario(t, "test/scenarios/samples/best_practices/disallow_root_user.yaml")
-}
-
```

Comment:
I compared `/samples/best_pratices` with `policies/best-practices`, it looks like some of the tests will not be covered in the new test. Can we please make sure all the old tests in kyverno/kyverno are properly migrated and covered by the test command?